### PR TITLE
SDIT-2044 Initial skeleton contact person resource

### PR DESCRIPTION
**Pull request by andymarke**
Also deleted some seed test files that didn't seem to be used anywhere

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/LegacyPersonBuilder.kt [hunk 1]
```diff
@@ -3,7 +3,7 @@ package uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Person
 import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.PersonPhone
 
-class PersonBuilder(
+class LegacyPersonBuilder(
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/LegacyPersonBuilder.kt:6]: renamed so it doesn't conflict with DSL builder - visit tests where never converted to use DSL

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResponseTest.kt [hunk 2]
```diff
@@ -269,10 +269,8 @@ internal class VisitResponseTest {
...
 }
 
-fun anOffenderBooking(nomsId: String): OffenderBooking {
-  return OffenderBookingBuilder().build(
-    offender = LegacyOffenderBuilder(nomsId = nomsId).build(Gender("F", "FEMALE")),
-    bookingSequence = 1,
-    AgencyLocation(id = "LEI", description = "Leeds HMP"),
-  )
-}
+fun anOffenderBooking(nomsId: String): OffenderBooking = OffenderBookingBuilder().build(
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResponseTest.kt:272]: ktlint

File: src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResourceIntTest.kt [hunk 3]
```diff
@@ -171,7 +171,7 @@ class VisitResourceIntTest : IntegrationTestBase() {
     fun `create visit with invalid person`() {
       val error = webTestClient.post().uri("/prisoners/$offenderNo/visits")
         .headers(setAuthorisation(roles = listOf("ROLE_NOMIS_VISITS")))
-        .body(BodyInserters.fromValue(createVisitWithPeople().copy(visitorPersonIds = listOf(-7L, -99L))))
+        .body(BodyInserters.fromValue(createVisitWithPeople().copy(visitorPersonIds = listOf(threePeople[0].id, -99L))))
         .exchange()
```
Comment (andymarke) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/visits/VisitResourceIntTest.kt:175]: -7 was seed data, so switched to person that must exist

**Review by lbennett-moj — APPROVED** [review]
lgtm

**Pull request merged**